apps/complaints/translations/src/en/validation.json

```diff
@@ -0,0 +1,5 @@
+{
+  "who-radio": {
+    "required": "Tell us who you are"
```

Comment:
this file shouldn't really exist - it would be nicer if they were together with the other field translations, not sure if the architecture allows at the mo, just a point to note.